extensions/remote-udp/README.md

```diff
@@ -0,0 +1,67 @@
+# Remote UDP Extension
+
+The openEO API is a specification for interoperable cloud-based processing of large Earth observation datasets.
+
+This extension enables user to load user-defined processes that are hosted external to the openEO API (e.g., GitHub or cloud storage) through the process namespace into process graphs.
+
+- Version: **0.1.0**
+- Stability: **experimental**
+- Conformance class: `https://api.openeo.org/extensions/remote-udp/0.1.0`
+
+## Justification
+
+The openEO API defines the `namespace` property in a process node of a process graph as follows:
+>  The following options are predefined by the openEO API, but additional namespaces may be introduced by back-ends or in a future version of the API.
+> * `null` [...]
+> * `backend` [...]
+> * `user` [...]
+
+This makes it possible for this extension to add additional allowed values to the `namespace` property.
+
+## Specification
+
+This extension extends the `namespace` property of process graph nodes so that it accepts **absolute** URL with the protocols `https` (**recommended**) and `http` (discouraged). The URLs specified MUST return one of the following two options:
+
+1. A single process, compatible to the endpoint `GET /process_graphs/{process_graph_id}`.
+   In this case, the `id` property of the process graph node MUST be equal to the `id` of the process, 
+   otherwise a `ProcessNamespaceInvalid` error is thrown
+2. A list of processes, compatible to the endpoint `GET /process_graphs`.
+   In this case, the `id` property of the process graph node is used to identify the process from the list.
+   If not found a `ProcessNamespaceInvalid` error is thrown
+
+### Compatibility
+
+Compatible means in this context that the requests and responses must comply to the openEO API specification with the following exceptions:
+
+- The `Authorization` header MUST NOT not be sent.
+- Lists of processes MUST NOT paginate and the full process description MUST be provided for each process (i.e., the recommendation to omit large properties such as `process_graph` doesn't apply).
```

Comment:
Doesn't this lead to a conflicting situation? If you list a lot of proceses *and* are forced to include the full process description, you are going to want to paginate this?

In any case, the fact that the process graph MUST be included should be move to the specification section I think, this is a hard requirement not just a compatibility  concern.

I think pagination is fine as long as the URL points to the right page directly so that the target process is included in the page. But that is kind of implicit from the current spec "`id` ... is used to identify the process from the list." Maybe what you want to note under this Compatibility section is that the backend is not expected to walk the pages (ie multiple requests) to find the process?